spec/facade/food_facade_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe FoodFacade do
+  it '#search food by keyword' do
+    results = FoodFacade.food_search("sweet potatoes")
+
```

Comment:
Similar to the Service, we can just test for data types here vs actual info. But nice job testing.